packages/ui-components/src/components/range/range.helper.ts

```diff
@@ -1,8 +1,9 @@
-export const getInputPercentageFromValue = (inputValue: string, min: number, max: number) => {
-	return (100 * (parseInt(inputValue) - min)) / (max - min);
+export const getInputPercentageFromValue = (inputValue: number, min: number, max: number) => {
+	return (100 * (inputValue - min)) / (max - min);
 };
 
 export const getOffset = (percentage: number) => {
-	const invertPercentage = 1 - percentage / 100;
-	return invertPercentage * 10 - 4;
+	const offSet = -2.5;
+
+	return (percentage / 10 - 5) * offSet;
```

Comment:
same for this `10 - 5`

Reply:
I needed to calculate the distance between the half of the slider and the button, that's why I subtract 5. I made it more clear in my newest commit.